packages/form-builder/src/components/sidebar/primary.js

```diff
@@ -40,6 +42,17 @@ const tabs = [
 
 function Sidebar() {
 
+    const [selectedTab, setSelectedTab] = useState(null);
+
+    const selectedBlocks = useSelectedBlocks();
```

Comment:
@jonwaldstein I pushed a refactor to extract a `useSelectedBlocks` hook. See https://github.com/impress-org/givewp-next-gen/pull/43/commits/3dc2f5cd075e263b0f1643c3f997efba54edb68f